pipelines/error_rate_pipeline_extract.py

```diff
@@ -0,0 +1,66 @@
+'''
+Extract step for application error rates pipeline
+'''
+import luigi
+import datetime
+import pathlib
+from os.path import dirname
+from pipeline_steps.production_error_rates_source import ProductionErrorRatesSource
+from pipeline_util.file_name_utility import FileNameUtility
+from pipeline_util.time_period import previous_month, DateRange
+
+
+class ErrorRatePipelineExtract(luigi.Task):
+    date_interval = luigi.DateIntervalParameter(
+        default=None,
+        always_in_help=True,
+        description=("Date interval of report. Defaults to last month.")
+    )
+
+    application = luigi.Parameter(
+        default='whitehall-admin',
+        always_in_help=True,
+        description=("Application to extract error rates for"))
```

Comment:
Just a formatting issue, but putting the last bracket on a newline would be more consistent.

Reply:
Cool - I'll fix this in a future PR since I've got into a bit of a mess with branches on branches.